bin/connectors.py

```diff
@@ -1611,21 +1620,23 @@ def getPosition(self)->dict:
                             gpsposition["bearing"] = float(position["bearing"])
                             gpsposition["speed_mph"] = float(position["speed_mph"])
                             gpsposition["isvalid"] = True
+                            gpsposition["source"] = "gps"
 
                             # sanity check
                             if gpsposition["latitude"] == 0 or gpsposition["longitude"] == 0:
```

Comment:
Is comparing to zero a valid sanity check? Yeah, I know we're not going to have any flights go to the equator or the prime meridian. What I'm really asking is whether the device actually has a "bad data" value vs assuming that zero means the data is bad.

Reply:
This portion of the code is executed when there is a 3D fix from the GPS.  So there should be a reasonable assumption that we have a valid GPS position at this point.  Part of my paranoia here is in the past we’ve had some APRS beacons that would transmit their GPS coords as 0.000,0.000.  So just extending that paranoia into double checking the lat/lon here.  Honestly, we shouldn’t ever have location exactly on 0,0…right?  And if we do, then what are we doing in the Atlantic ocean off the west coast of Africa???  Hehe

I don't think checking for 0's in this case is impactful to any real locations being used.

-JD